src/style.css

```diff
@@ -1 +1,65 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  src: url('./fonts/Avenir-Heavy.ttf');
+  src: url('./fonts/Avenir.ttc');
+  font-style: normal;
+  font-weight: normal;
+  color: rgba(61, 78, 97, 1);
+}
+
+* {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
it is better to set font-family for html tag selector and let other elements inherit it